File: Freestyle/Webservice/WcfService/Models/DTO/ToDb/ReqRhyme.cs
namespace WcfService.Models.DTO.ToDb
{
    using System.Runtime.Serialization;

    [DataContract]
    public class ReqRhyme
    {
        [DataMember]
        public string UserKey{ get; set;}

        [DataMember]
        public int RhymeId{ get; set; }

        [DataMember]
        public string CurValue { get; set; }

        [DataMember]
        public bool IsNext { get; set; }
    }
}

File: Freestyle/Android/Freestyle/src/freestyle/data/dto/todb/ReqRhyme.java
package freestyle.data.dto.todb;

public class ReqRhyme {
	public ReqRhyme(String userKey, int rhymeId, String curValue, boolean isNext){
		this.userKey = userKey;
		this.rhymeId = rhymeId;
		this.curValue = curValue;
		this.isNext = isNext;
	}
	public String userKey;
	public int rhymeId;
	public String curValue;
	public boolean isNext;
}
